compiler/rustc_hir_typeck/src/lib.rs

```diff
@@ -167,6 +167,11 @@ fn typeck_with_fallback<'tcx>(
 
         let expected_type = fcx.normalize(body.value.span, expected_type);
 
+        // FIXME error tainting <https://github.com/rust-lang/rust/issues/129952>
```

Comment:
```suggestion
        // FIXME(#129952): We probably want a more principled approach here.
```

Reply:
mostly just the `FIXME(#129952)` I don't care too much about the actual fixme :blush: